CRITICAL - 2022-02-25 04:00:14 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(132): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 07:51:47 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::insert_id()
#0 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(96): App\Models\FinanceModel->save_item('expense', Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#2 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-25 07:54:13 --> Undefined variable: data
#0 /Applications/MAMP/htdocs/zlatar_finance/app/Models/FinanceModel.php(164): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/Applications/M...', 164, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(124): App\Models\FinanceModel->save_itemDoc('expense', 39)
#2 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#3 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-02-25 07:55:57 --> Argument 1 passed to CodeIgniter\Database\BaseBuilder::update() must be of the type array or null, string given, called in /Applications/MAMP/htdocs/zlatar_finance/app/Models/FinanceModel.php on line 164
#0 /Applications/MAMP/htdocs/zlatar_finance/app/Models/FinanceModel.php(164): CodeIgniter\Database\BaseBuilder->update('expense', Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(124): App\Models\FinanceModel->save_itemDoc('expense', 40, Array)
#2 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#3 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2022-02-25 07:57:35 --> Cannot add or update a child row: a foreign key constraint fails (`zlatar_finance`.`expense`, CONSTRAINT `expense_related_user` FOREIGN KEY (`author`) REFERENCES `users` (`id`))
CRITICAL - 2022-02-25 07:57:35 --> Cannot add or update a child row: a foreign key constraint fails (`zlatar_finance`.`expense`, CONSTRAINT `expense_related_user` FOREIGN KEY (`author`) REFERENCES `users` (`id`))
#0 /Applications/MAMP/htdocs/zlatar_finance/system/Database/MySQLi/Connection.php(292): mysqli->query('REPLACE INTO `e...', 0)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('REPLACE INTO `e...')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('REPLACE INTO `e...')
#3 /Applications/MAMP/htdocs/zlatar_finance/system/Database/BaseBuilder.php(1837): CodeIgniter\Database\BaseConnection->query('REPLACE INTO `e...', Array, false)
#4 /Applications/MAMP/htdocs/zlatar_finance/app/Models/FinanceModel.php(164): CodeIgniter\Database\BaseBuilder->replace(Array)
#5 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(124): App\Models\FinanceModel->save_itemDoc('expense', 41, Array)
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#7 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-25 09:40:53 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 10:04:03 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 10:13:16 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 10:15:02 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 10:20:20 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 10:20:47 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 10:21:34 --> fopen(http://localhost/zlatar_finance/public/uploads/index.html): failed to open stream: HTTP wrapper does not support writeable connections
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://lo...', '/Applications/M...', 179, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(179): fopen('http://localhos...', 'x+b')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/HTTP/Files/UploadedFile.php(130): CodeIgniter\HTTP\Files\UploadedFile->setPath('http://localhos...')
#3 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(129): CodeIgniter\HTTP\Files\UploadedFile->move('http://localhos...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-25 13:00:36 --> syntax error, unexpected '='
#0 /Applications/MAMP/htdocs/zlatar_finance/system/Autoloader/Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('/Applications/M...')
#1 /Applications/MAMP/htdocs/zlatar_finance/system/Autoloader/Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-02-25 13:22:02 --> Cannot add or update a child row: a foreign key constraint fails (`zlatar_finance`.`receipt`, CONSTRAINT `receipt_related_agreemnt` FOREIGN KEY (`agreement`) REFERENCES `agreement_forZR` (`id`))
CRITICAL - 2022-02-25 13:22:02 --> Cannot add or update a child row: a foreign key constraint fails (`zlatar_finance`.`receipt`, CONSTRAINT `receipt_related_agreemnt` FOREIGN KEY (`agreement`) REFERENCES `agreement_forZR` (`id`))
#0 /Applications/MAMP/htdocs/zlatar_finance/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `re...', 0)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `re...')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `re...')
#3 /Applications/MAMP/htdocs/zlatar_finance/system/Database/BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 /Applications/MAMP/htdocs/zlatar_finance/app/Models/FinanceModel.php(152): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(146): App\Models\FinanceModel->save_item('receipt', Array)
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#7 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-25 13:48:31 --> Undefined variable: title
#0 /Applications/MAMP/htdocs/zlatar_finance/app/Views/dashboard/finance_report.php(4): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/Applications/M...', 4, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/View/View.php(210): include('/Applications/M...')
#2 /Applications/MAMP/htdocs/zlatar_finance/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /Applications/MAMP/htdocs/zlatar_finance/system/Common.php(1089): CodeIgniter\View\View->render('dashboard/finan...', Array, true)
#4 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(150): view('dashboard/finan...', Array)
#5 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#6 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#7 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-25 13:51:30 --> Undefined variable: userInfo
#0 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(145): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/Applications/M...', 145, Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->save_item()
#2 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-25 13:52:29 --> Call to undefined method App\Models\FinanceModel::get_all_recipt_items
#0 /Applications/MAMP/htdocs/zlatar_finance/app/Controllers/Dashboard.php(38): CodeIgniter\Model->__call('get_all_recipt_...', Array)
#1 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(825): App\Controllers\Dashboard->get_receipt_items()
#2 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 /Applications/MAMP/htdocs/zlatar_finance/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Applications/MAMP/htdocs/zlatar_finance/public/index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
